File: Code coach/Medium/Secret message/secret_message.cpp
#include <iostream>
using namespace std;

int main()
{
	string text;
	getline(cin, text);

	string textInvertedAscii = "";
	for (auto c : text) 
	{
		if (isalpha(c))
		{
			int asciiLowA = 97;
			int asciiLowZ = 122;
			textInvertedAscii += char(asciiLowZ - (int) tolower(c) + asciiLowA);
		} 
		else 
		{
			textInvertedAscii += c;
		}
	}
 
	cout << textInvertedAscii;

	return 0;
}

File: Code coach/Medium/Secret message/secret_message.py
def backWard_char(char):
  if char.isalpha():
    ascii_low_a = 97
    ascii_low_z = 122
    return chr(ascii_low_z - ord(char.lower()) + ascii_low_a)
  return char

text = input()
text_inverted_ascii = ''.join(map(backWard_char, text))

print(text_inverted_ascii)
